test/where_clauses_simple/src/main.rs

```diff
@@ -0,0 +1,26 @@
+trait Ops {
```

Comment:
An interesting example (which I don't think you handle yet) is supertraits (or "traits with parents" to reuse Charon's terminology).
Ex.:
```rust
trait T1 { ... }
trait T2 : T1 { ...}
```

Those happen in many places in Rust with `Copy`/`Clone`, `Eq`/`PartialEq`, `Ord`/`PartialOrd`, etc.

Reply:
Yes these are in the (disabled) where_clauses trait.